code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -83,26 +83,21 @@
 
 //////////////HOSTILE MOB TARGETTING AND AGGRESSION////////////
 
-
 /mob/living/simple_animal/hostile/proc/ListTargets()//Step 1, find out what we can see
 	. = list()
 	if(!search_objects)
 		var/list/Mobs = hearers(vision_range, targets_from) - src //Remove self, so we don't suicide
```

Comment:
probably some ancient rumour about ```hearers()``` being slightly more performant? something spooky like that probably.